math/R-app/Portfile

```diff
@@ -1,10 +1,17 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
-PortGroup xcode 1.0
+PortSystem              1.0
+PortGroup               xcode 1.0
 
 name                    R-app
-version                 1.78
+
+# Update for newer systems, keep pegged for older. Compatibility see in R.xcodeproj/project.pbxproj
+if {${os.platform} eq "darwin" && ${os.major} > 12} {
+    version             1.78
+} else {
+    # Version for older systems, including PPC. 1.73 still builds but freezes on quit.
+    version             1.72
+}
```

Comment:
You're also used version `1.70` for everything before 11. Maybe keep versions at one place?